homeassistant/components/velbus/sensor.py

```diff
@@ -90,3 +90,13 @@ def state_class(self):
         if self._is_counter:
             return STATE_CLASS_TOTAL_INCREASING
         return STATE_CLASS_MEASUREMENT
+
+    @property
+    def extra_state_attributes(self):
+        """Return the state attributes of the sun."""
+        if self._channel.is_temperature():
+            return {
+                "Max": self._channel.get_max(),
```

Comment:
All measurements that are valid as separate sensors should be separate sensors. A max temperature can be a separate sensor.

Reply:
Please don't add new feature besides the changes that are required for the new library. Other changes should go in separate PRs. Thanks!